marking-criteria.qmd

```diff
@@ -1,19 +1,20 @@
-
 ---
 title: "Marking Criteria"
-format: html
+format:
+  html:
+    theme: cerulean
```

Comment:
Shouldn't we apply the them site-wide?

Reply:
Yes, it makes sense to stick to one theme site-wide. This is a test at the moment. I think we can first identify the theme we like best and then apply it site-wide. There are many to choose from: https://bootswatch.com

Reply:
Great plan.

Reply:
Marking criteria looking good here: https://itsleeds.github.io/TDStests/marking-criteria.html